system/Database/SQLSRV/Forge.php

```diff
@@ -144,7 +144,25 @@ protected function _alterTable(string $alterType, string $table, $field)
                 }
             }
 
-            $sql = 'ALTER TABLE ' . $this->db->escapeIdentifiers($this->db->schema) . '.' . $this->db->escapeIdentifiers($table) . ' DROP ';
+            $fullTable = $this->db->escapeIdentifiers($this->db->schema) . '.' . $this->db->escapeIdentifiers($table);
+
+            // Drop default constraints
+            $fields = "'" . implode("','", (array) $field) . "'";
```

Comment:
Should use `escapeIdentifiers()` to field.

Reply:
Its not being used as an identifier but is a literal string.

Reply:
I changed to `escape()`